src/libraries/System.Text.Json/Common/JsonConstants.cs

```diff
@@ -0,0 +1,11 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+namespace System.Text.Json
+{
+    internal static partial class JsonConstants
```

Comment:
nit\future: if this list grows much, we may want a shared JsonConstancts.cs file.

Reply:
This _is_ the shared file.